superset/databases/ssh_tunnel/commands/update.py

```diff
@@ -33,6 +34,9 @@
 
 logger = logging.getLogger(__name__)
 
+config = app.config
+ssh_tunnel_manager = config["SSH_TUNNEL_MANAGER"]
```

Comment:
Same comment as before. We should be getting the `SSHManager` from the `current_app`